### PR TITLE
Minor improvements to `manage` and `api`, add latest commit ID section in `get_device_info`

### PR DESCRIPTION
**Pull request by cycool29**
- Remove `-c` flag in aria2_flags for `--allow-overwrite` to work
- Remove dupicate `format_logfile`
- Add latest commit ID section in `get_device_info` to check if user's pi-apps is even with repo's one
- cd to $HOME in api
- Show `Download url to path` if not download as `$(pwd)/$(basename $url)` in `wget`

### CODE REVIEW

File: manage [hunk 1]
```diff
@@ -272,7 +271,7 @@ Or on Discord: \e[94m\e[4mhttps://discord.gg/RXSTvaUvuu\e[0m" | tee -a "$logfile
     fi
     
     format_logfile "$logfile" #remove escape sequences from logfile
-    mv "$logfile" "$(echo "$logfile" | sed 's+-incomplete-+-fail-+g')"
+    mv "$logfile" "$(echo "$logfile" | sed 's+-incomplete-+-fail-+g')" && logfile="$(echo "$logfile" | sed 's+-incomplete-+-fail-+g')"
```
Comment (Botspot) [inline manage:274]: What effect does changing the value of the `$logfile` variable do? The `manage` script exits shortly afterwards.

Comment (cycool29) [reply manage:274]: The logfile still needed by `log_diagnose`.

Comment (cycool29) [reply manage:274]: image

Comment (Botspot) [reply manage:274]: Looks like we're dealing with duplicate code. There's no need to run `log_diagnose` twice in `manage install`.

**Pull request merged**